roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -102,7 +102,7 @@
     - kubeadm_already_run.stat.exists
 
 - name: kubeadm | Initialize first master
-  command: timeout -k 600s 600s {{ bin_dir }}/kubeadm init --config={{ kube_config_dir }}/kubeadm-config.yaml --ignore-preflight-errors=all
+  command: timeout -k 600s 600s {{ bin_dir }}/kubeadm init --config={{ kube_config_dir }}/kubeadm-config.yaml --ignore-preflight-errors=all --skip-phases=addon/kube-proxy
```

Comment:
`--skip-phases=addon/kube-proxy` is missing in task: `kubeadm | Init other uninitialized masters`

Reply:
I think we need to skip the phase too on upgrades in https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubernetes/master/tasks/kubeadm-upgrade.yml

Reply:
good catch

Reply:
Still missing in `kubeadm | Init other uninitialized masters`